<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtShowcase</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="hlayTop">
      <item>
       <widget class="QLabel" name="lblStyle">
        <property name="text">
         <string>Style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbStyle">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hspTop">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="chDisable">
        <property name="text">
         <string>Disable Widgets</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="mainWidget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="gbRadios">
         <property name="title">
          <string>Radios</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="rbTurtles">
            <property name="text">
             <string>I like turtles</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbTrains">
            <property name="text">
             <string>I like trains</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbTooltip">
            <property name="toolTip">
             <string>trains are cooler</string>
            </property>
            <property name="text">
             <string>I have a tooltip</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chTristate">
            <property name="text">
             <string>Tristate</string>
            </property>
            <property name="tristate">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vspRadios">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>2</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="gbButtons">
         <property name="title">
          <string>Buttons</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pbDefault">
            <property name="text">
             <string>Default Pushbutton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbToggle">
            <property name="text">
             <string>Toggle Pushbutton</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbFlat">
            <property name="text">
             <string>Flat Pushbutton</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vspButtons">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="twWidgets">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tabTable">
          <attribute name="title">
           <string>Table</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <row>
              <property name="text">
               <string>Peter</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Harry</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Kimi</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Max</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Surname</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Age</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Hobby</string>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string>Parker</string>
              </property>
             </item>
             <item row="0" column="1">
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item row="0" column="2">
              <property name="text">
               <string>Swoosh</string>
              </property>
             </item>
             <item row="1" column="0">
              <property name="text">
               <string>Botter</string>
              </property>
             </item>
             <item row="1" column="1">
              <property name="text">
               <string>42</string>
              </property>
             </item>
             <item row="1" column="2">
              <property name="text">
               <string>Scary Harry</string>
              </property>
             </item>
             <item row="2" column="0">
              <property name="text">
               <string>Räikkönen</string>
              </property>
             </item>
             <item row="2" column="1">
              <property name="text">
               <string>41</string>
              </property>
             </item>
             <item row="2" column="2">
              <property name="text">
               <string>Talking</string>
              </property>
             </item>
             <item row="3" column="0">
              <property name="text">
               <string>Musterman</string>
              </property>
             </item>
             <item row="3" column="1">
              <property name="text">
               <string>22</string>
              </property>
             </item>
             <item row="3" column="2">
              <property name="text">
               <string>Cloning</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabText">
          <attribute name="title">
           <string>Text Edit</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="gbInputs">
         <property name="title">
          <string>Inputs</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="leUsername">
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lePassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFontComboBox" name="fcbFonts"/>
          </item>
          <item>
           <widget class="QSpinBox" name="sbNumber">
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dteDatetime"/>
          </item>
          <item>
           <widget class="QKeySequenceEdit" name="kseShortcuts"/>
          </item>
          <item>
           <widget class="QSlider" name="slSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vspInputs">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToolbar"/>
    <addaction name="actionStatusbar"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionShowcase"/>
    <addaction name="actionIcons8"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionToolbar"/>
   <addaction name="actionStatusbar"/>
   <addaction name="separator"/>
   <addaction name="actionShowcase"/>
   <addaction name="actionIcons8"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/open.png</normaloff>:/resources/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/save.png</normaloff>:/resources/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/export.png</normaloff>:/resources/icons/export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/quit.png</normaloff>:/resources/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/copy.png</normaloff>:/resources/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/cut.png</normaloff>:/resources/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/paste.png</normaloff>:/resources/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/tool.png</normaloff>:/resources/icons/tool.png</iconset>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
   <property name="toolTip">
    <string>Toolbar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionStatusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/status.png</normaloff>:/resources/icons/status.png</iconset>
   </property>
   <property name="text">
    <string>Statusbar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionShowcase">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/icon.png</normaloff>:/resources/icons/icon.png</iconset>
   </property>
   <property name="text">
    <string>Showcase</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionIcons8">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/icons8.png</normaloff>:/resources/icons/icons8.png</iconset>
   </property>
   <property name="text">
    <string>Icons8</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
